engine/jsb-sys.js

```diff
@@ -46,6 +46,9 @@ sys.getSafeAreaRect = function() {
     var relatedPos = {left: 0, top: 0, width: screenSize.width, height: screenSize.height};
     cc.view.convertToLocationInView(leftBottom.x, leftBottom.y, relatedPos, leftBottom);
     cc.view.convertToLocationInView(rightTop.x, rightTop.y, relatedPos, rightTop);
+    // convert view point to canvas piexl size
```

Comment:
```suggestion
    // convert view point to canvas piexl size
```
补充一下 jare 说的，应该是 `画布像素分辨率` -> `设计分辨率`，参考下面的注释
```JavaScript
/**
     * !#en
     * Returns scale factor of the horizontal direction (X axis).
     * !#zh 返回横轴的缩放比，这个缩放比是将画布像素分辨率放到设计分辨率的比例。
     * @method getScaleX
     * @return {Number}
     */
    getScaleX: function () {
        return this._scaleX;
    },
```